client/src/components/FeedbackForm/FeedbackButton.scss

```diff
@@ -0,0 +1,9 @@
+.feedback-button {
+  align-self: center;
+  position: absolute;
+  bottom: 5px;
+  .btn {
+    font-size: 0.75em;
+    font-weight: 500;
+  }
```

Comment:
### Hint

When you migrate the styles of this page to CSS Module stylesheets, this could be `.button` (via `<Button styleName="button" … >`).